### DISCUSSION TITLE
ToonSquid Mac Support

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#407] [Ideas]
I've been hearing that Mac devices can run iOS and iPadOS apps. I wonder if anyone got ToonSquid to run on a Mac Mini, and if so, I wonder if it would work with a Wacom tablet, because I have a Wacom One, the one with the screen, and if I got a Mac Mini, I could probably use it with the Wacom One, run ToonSquid, and use it as a high quality animation app on computer.

**Comment by Cat-Butler** [comment]
See [Keiwan's earlier reply](https://github.com/keiwando/toonsquid/discussions/57#discussioncomment-4147477) to the same question.
